worker/worker_test.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/dadosjusbr/alba/storage"
+	"github.com/dadosjusbr/executor"
+	"github.com/matryer/is"
+)
+
+var pipelinesDB = []storage.Pipeline{
```

Comment:
Tu deve precisar criar vários desses a medida que o teste aumenta.

Sugestão: cria os `storage.Pipeline` baseado nas características que você quer, coloca nomes sugestivos neles (dada as características de erro ou algo simples ou complexo ou algo assim) e coloca eles na parte de baixo do programa.

Daí, cria o `[]storage.Pipeline` dentro de cada método. A pessoa lendo esse arquivo ver os casos de teste e quando for ler um terminado método vai entender mais fácilmente o caso sendo testado.